2021-05-10 17:58:36,554 |main:28|[WARNING]: Update "None" caused error "Conflict: terminated by other getUpdates request; make sure that only one bot instance is running"
2021-05-10 17:58:41,666 |main:28|[WARNING]: Update "None" caused error "Conflict: terminated by other getUpdates request; make sure that only one bot instance is running"
2021-05-10 18:26:24,661 |main:29|[WARNING]: Update "{'update_id': 916158064, 'message': {'message_id': 36, 'date': 1620660384, 'chat': {'id': 367458916, 'type': 'private', 'username': 'sah4ez', 'first_name': 'Alexandr', 'last_name': 'Kozlenkov'}, 'text': '/optimize nsga3 1 1', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 9}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 367458916, 'first_name': 'Alexandr', 'is_bot': False, 'last_name': 'Kozlenkov', 'username': 'sah4ez', 'language_code': 'en'}}}" caused error "'CallbackContext' object has no attribute 'groups'"
2021-05-10 18:28:41,321 |main:29|[WARNING]: Update "{'update_id': 916158065, 'message': {'message_id': 38, 'date': 1620660520, 'chat': {'id': 367458916, 'type': 'private', 'username': 'sah4ez', 'first_name': 'Alexandr', 'last_name': 'Kozlenkov'}, 'text': '/optimize nsga3 1 1', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 9}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 367458916, 'first_name': 'Alexandr', 'is_bot': False, 'last_name': 'Kozlenkov', 'username': 'sah4ez', 'language_code': 'en'}}}" caused error "expected string or bytes-like object"
2021-05-10 18:29:50,089 |main:29|[WARNING]: Update "{'update_id': 916158066, 'message': {'message_id': 40, 'date': 1620660589, 'chat': {'id': 367458916, 'type': 'private', 'username': 'sah4ez', 'first_name': 'Alexandr', 'last_name': 'Kozlenkov'}, 'text': '/optimize nsga3 1 1', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 9}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 367458916, 'first_name': 'Alexandr', 'is_bot': False, 'last_name': 'Kozlenkov', 'username': 'sah4ez', 'language_code': 'en'}}}" caused error "'CallbackContext' object has no attribute 'text'"
2021-05-10 18:30:20,922 |main:29|[WARNING]: Update "{'update_id': 916158067, 'message': {'message_id': 42, 'date': 1620660620, 'chat': {'id': 367458916, 'type': 'private', 'username': 'sah4ez', 'first_name': 'Alexandr', 'last_name': 'Kozlenkov'}, 'text': '/optimize nsga3 1 1', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 9}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 367458916, 'first_name': 'Alexandr', 'is_bot': False, 'last_name': 'Kozlenkov', 'username': 'sah4ez', 'language_code': 'en'}}}" caused error "'Update' object has no attribute 'text'"
2021-05-10 18:30:50,937 |main:29|[WARNING]: Update "{'update_id': 916158068, 'message': {'message_id': 44, 'date': 1620660650, 'chat': {'id': 367458916, 'type': 'private', 'username': 'sah4ez', 'first_name': 'Alexandr', 'last_name': 'Kozlenkov'}, 'text': '/optimize nsga3 1 1', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 9}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 367458916, 'first_name': 'Alexandr', 'is_bot': False, 'last_name': 'Kozlenkov', 'username': 'sah4ez', 'language_code': 'en'}}}" caused error "'NoneType' object has no attribute 'groups'"
2021-05-10 18:31:53,300 |main:29|[WARNING]: Update "{'update_id': 916158069, 'message': {'message_id': 46, 'date': 1620660712, 'chat': {'id': 367458916, 'type': 'private', 'username': 'sah4ez', 'first_name': 'Alexandr', 'last_name': 'Kozlenkov'}, 'text': '/optimize nsga3 1 1', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 9}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 367458916, 'first_name': 'Alexandr', 'is_bot': False, 'last_name': 'Kozlenkov', 'username': 'sah4ez', 'language_code': 'en'}}}" caused error "'NoneType' object has no attribute 'group'"
2021-05-10 18:36:11,279 |main:29|[WARNING]: Update "{'update_id': 916158070, 'message': {'message_id': 48, 'date': 1620660970, 'chat': {'id': 367458916, 'type': 'private', 'username': 'sah4ez', 'first_name': 'Alexandr', 'last_name': 'Kozlenkov'}, 'text': '/optimize nsga3 1 1', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 9}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 367458916, 'first_name': 'Alexandr', 'is_bot': False, 'last_name': 'Kozlenkov', 'username': 'sah4ez', 'language_code': 'en'}}}" caused error "name 'regex' is not defined"
2021-05-10 18:38:48,896 |main:29|[WARNING]: Update "{'update_id': 916158073, 'message': {'message_id': 54, 'date': 1620661128, 'chat': {'id': 367458916, 'type': 'private', 'username': 'sah4ez', 'first_name': 'Alexandr', 'last_name': 'Kozlenkov'}, 'text': '/optimize nsga3 1 1', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 9}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 367458916, 'first_name': 'Alexandr', 'is_bot': False, 'last_name': 'Kozlenkov', 'username': 'sah4ez', 'language_code': 'en'}}}" caused error "'NoneType' object has no attribute 'group'"
2021-05-10 19:14:28,744 |main:31|[WARNING]: Update "{'update_id': 916158076, 'message': {'message_id': 61, 'date': 1620663268, 'chat': {'id': 367458916, 'type': 'private', 'username': 'sah4ez', 'first_name': 'Alexandr', 'last_name': 'Kozlenkov'}, 'text': '/price', 'entities': [{'type': 'bot_command', 'offset': 0, 'length': 6}], 'caption_entities': [], 'photo': [], 'new_chat_members': [], 'new_chat_photo': [], 'delete_chat_photo': False, 'group_chat_created': False, 'supergroup_chat_created': False, 'channel_chat_created': False, 'from': {'id': 367458916, 'first_name': 'Alexandr', 'is_bot': False, 'last_name': 'Kozlenkov', 'username': 'sah4ez', 'language_code': 'en'}}}" caused error "not enough arguments for format string"
